model_types/modelTypesList.json

```diff
@@ -0,0 +1,90 @@
+{
+  "modelTypes": [
+    {
+      "modelType": "synthetics",
+      "defaultConfig":  "config_templates/gretel/synthetics/tabular-lstm-evaluate.yml",
+      "sampleDataset": {
+        "fileName": "bank_marketing_small.csv",
+        "description": "Create synthetic data based on the publicly available dataset predicting opting in or out of bank marketing.",
+        "records": 4521,
+        "fields": 17,
+        "trainingTime": "< 10 mins",
+        "bytes": 371020
+      }
+    },
+    {
+      "modelType": "gpt_x",
+      "defaultConfig": "config_templates/gretel/synthetics/natural-language.yml",
+      "sampleDataset": {
+        "fileName": "taylor-swift-lyrics.csv",
+        "description": "Create believable song lyrics using this dataset of Taylor Swift hits.",
+        "records": 246,
+        "fields": 1,
+        "trainingTime": "< 5 mins",
+        "bytes": 405000
+      }
+    },
+    {
+      "title": "ACTGAN",
+      "modelType": "actgan",
+      "defaultConfig": "config_templates/gretel/synthetics/tabular-actgan.yml",
+      "sampleDataset": {
+        "fileName": "sample-synthetic-healthcare.csv",
+        "description": "Use this sample electronic health records (EHR) dataset to synthesize an entirely new set of statistically equivalent records.",
+        "records": 9999,
+        "fields": 18,
+        "trainingTime": "6 mins",
+        "bytes": 830021
+      }
+    },
+    {
+      "title": "Amplify",
```

Comment:
Can we add Titles for all the model types instead of just a few?

Reply:
Per my other comment - we actually already have all the 'titles' (called labels in console) tied to the model type - I will clean this up to remove all titles from here

Reply:
sounds good, thank you